src/Entity/Mooc/BaseMoocElement.php

```diff
@@ -3,20 +3,28 @@
 namespace AppBundle\Entity\Mooc;
 
 use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
-use AppBundle\AppBundle;
 use AppBundle\Entity\EntityTimestampableTrait;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
 use Gedmo\Mapping\Annotation as Gedmo;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 use Symfony\Component\Validator\Constraints as Assert;
 
 /**
  * @ORM\Entity
- * @ORM\Table(name="mooc_elements")
+ * @ORM\Table(
+ *     name="mooc_elements",
+ *     uniqueConstraints={@ORM\UniqueConstraint(name="mooc_chapter_video_slug", columns="slug")}
```

Comment:
`name="mooc_element_slug"`

What about adding the chapter unique constraint here ?